dbservice/postgresdb/actions.go

```diff
@@ -14,7 +14,6 @@ func (postgresDb *PostgresDb) MayBeStoreMessage(message []byte, messageKey strin
 	if err != nil {
 		return false, err
 	}
-	defer db.Close()
```

Comment:
Is there a terminate function we can add it to?

Reply:
We can expose an API for that, and add it to the main terminate func